cmd/dexctl/main.go

```diff
@@ -53,7 +56,10 @@ var (
 func init() {
 	log.EnableTimestamps()
 
-	rootCmd.PersistentFlags().StringVar(&global.dbURL, "db-url", "", "DSN-formatted database connection string")
+	rootCmd.PersistentFlags().StringVar(&global.dbURL, "db-url", "", "DSN-formatted database connection string. --db-url flag is deprecated. Use --base-url and --api-key")
+	rootCmd.PersistentFlags().StringVar(&global.baseURL, "base-url", "", "DSN-formatted dex-overlord base URL")
```

Comment:
probably don't need to specify the "DSN-formatting" maybe just "URL of the dex-overlord." Though we probably want to just add the well-known path of the API to the base url.